src/app/gallery.component.ts

```diff
@@ -1,34 +1,22 @@
-import { Component, ViewChild, ElementRef, AfterViewInit } from '@angular/core';
-import { fromEvent } from 'rxjs';
-import { bufferCount } from 'rxjs/operators';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
 
 @Component({
     selector: 'app-gallery',
-    templateUrl: './gallery.component.html'
+    templateUrl: './gallery.component.html',
+    styleUrls: ['./gallery.component.scss']
 })
 
-export class GalleryComponent implements AfterViewInit {
-    @ViewChild('btn', { static: false })
-    btn: ElementRef;
+export class GalleryComponent {
+    @Input()    
```

Comment:
recieve with this input array from app component, which contains images urls. it should be one variable, because input works only with one variable